content/blog/2025/02/2025-02-14-building-a-safer-matrix.md

```diff
@@ -190,40 +190,40 @@ Rechenschaftspflicht ist wichtig, deshalb wollen wir dieses Jahr mit der Veröff
 * Wenn Sie einen anderen Matrix-Server verwenden, können Sie die Meldung ebenfalls von Ihrem Client aus an Ihren Server-Administrator senden.  
-* Wenn Sie kein Matrix-Nutzer sind oder wenn Sie ein Nutzer auf einem anderen Homeserver sind und uns über schädliche Inhalte auf matrix.org informieren möchten, können Sie uns eine E-Mail mit den Informationen, die Sie haben, an abuse@matrix.org schicken. Raum- und Benutzerkennungen sind sehr hilfreich. Bitte schicken Sie uns keine Bildschirmfotos von schädlichen Inhalten \- wir werden Ihnen Bescheid geben, wenn wir weitere Informationen benötigen.
+* Wenn Sie kein Matrix-Nutzer sind oder wenn Sie ein Nutzer auf einem anderen Homeserver sind und uns über schädliche Inhalte auf Matrix.org informieren möchten, können Sie uns eine E-Mail mit den Informationen, die Sie haben, an [abuse@matrix.org](mailto:abuse@matrix.org) schicken. Raum- und Benutzerkennungen sind sehr hilfreich. Bitte schicken Sie uns keine Bildschirmfotos von schädlichen Inhalten \- wir werden Ihnen Bescheid geben, wenn wir weitere Informationen benötigen.
 
-### Als Matrix Server Betreiber
+### Als Matrix-Server-Betreiber
 
-* Die offene Registrierung ist in Synapse [standardmäßig deaktiviert](https://github.com/matrix-org/synapse/commit/58367a9da2539abdbfe4dc817fba5b179b95334b), und wir unterstützen diese Voreinstellung. Wenn Ihr Server eine offene Registrierung anbietet, müssen Sie in ein Sicherheitsteam investieren, um eine angemessene Moderation zu gewährleisten und die Risiken zu minimieren, die entstehen, wenn Sie unbekannten Benutzern die Nutzung Ihres Servers erlauben.  
+* Die offene Registrierung ist in Synapse [standardmäßig deaktiviert](https://github.com/matrix-org/synapse/commit/58367a9da2539abdbfe4dc817fba5b179b95334b), und wir befürworten diese Voreinstellung. Wenn Ihr Server eine offene Registrierung anbietet, **müssen** Sie in ein Trust & Safety Team investieren, um eine angemessene Moderation zu gewährleisten und die Risiken zu minimieren, die entstehen, wenn Sie unbekannten Benutzern die Nutzung Ihres Servers erlauben.  
 * Überprüfen Sie, wer sich auf Ihrem Server anmeldet und in welchen Räumen Ihr Server Mitglied ist.  
-* Überprüfen Sie die Berichte Ihrer Nutzer und ergreifen Sie Maßnahmen, um gemeldete schädliche Inhalte zu entfernen. Überprüfen Sie Ihre rechtlichen Verpflichtungen in dem Land, in dem Sie Ihren Server betreiben.  
+* Überprüfen Sie die Meldungen Ihrer Nutzer und ergreifen Sie Maßnahmen, um gemeldete schädliche Inhalte zu entfernen. Überprüfen Sie Ihre rechtlichen Verpflichtungen in dem Land, in dem Sie Ihren Server betreiben.  
```

Comment:
technical lingo, I think. matches "gemeldete"

Reply:
Also, I think "Berichte" in context would be more along the lines of some hearsay while "Meldung" has a sense of intent.